src/tests/ffi.cpp

```diff
@@ -5451,6 +5451,121 @@ TEST_F(rnp_tests, test_ffi_keys_import)
     rnp_ffi_destroy(ffi);
 }
 
+static std::vector<uint8_t>
+read_file_to_vector(const char *valid_key_path, size_t allowed_size)
+{
+    FILE *  fp;
+    size_t  read_len;
+    uint8_t pkt[allowed_size + 1];
+    // open file for binary reading mode
+    fp = fopen(valid_key_path, "rb");
+    assert_non_null(fp);
+    read_len = fread(pkt, 1, sizeof(pkt), fp);
+    assert_true(read_len <= allowed_size);
```

Comment:
assert() in helper function would not allow to know which test actually caused it, so it would be better to return true/false and check it on upper level.